app/_src/policies/introduction.md

```diff
@@ -1,6 +1,661 @@
+   ```
+
+There is however, one exception to this when using `MeshService` with **outbound** policies (policies with `spec.to[].targetRef`).
+In this case, if you define a policy in the same namespace as the `MeshService` it is defined in, that policy will be considered a **producer** policy.
+This means that all clients of this service (even in different zones) will be impacted by this policy.
+
+An example of a producer policy is:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: timeout-to-redis
+  namespace: kuma-demo
+spec:
+  to:
+  - targetRef:
+      kind: MeshService
+      name: redis
+    default:
+      connectionTimeout: 10s
+```
+
+The other type of policy is a consumer policy which most commonly use labels to match a service.
+
+An example of a consumer policy which would override the previous producer policy:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: timeout-to-redis-consumer
+  namespace: kuma-demo
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        labels:
+          k8s.kuma.io/service-name: redis
+      default:
+        connectionTimeout: 10s
+```
+
+{% tip %}
+Remember that `labels` on a `MeshService` applies to _each_ matching `MeshService`. To communicate to services
+named the same way in different namespaces or zones with different configuration use a more specific set of labels.
+{% endtip %}
+
+{{ site.mesh_product_name }} adds a label `kuma.io/policy-role` to identify the type of the policy. The values of the label are:
+
+- **system**: Policies defined on global or in the zone's system namespace
+- **workload-owner**: Policies defined in a non system namespaces that do not have `spec.to` entries, or have both `spec.from` and `spec.to` entries
+- **consumer**: Policies defined in a non system namespace that have `spec.to` which either do not use `name` or have a different `namespace`
+- **producer**: Policies defined in the same namespace as the services identified in the `spec.to[].targetRef`
+
+The merging order of the different policy scopes is: **global < zonal < producer < consumer < worload-owner**.
+
+### Example
+
+We have 2 clients client1 and client2 they run in different namespaces respectively ns1 and ns2.
+
+{% mermaid %}
+flowchart LR
+subgraph ns1
+    client1(client)
+end
+subgraph ns2
+  client2(client)
+  server(MeshService: server)
+end
+client1 --> server
+client2 --> server
+{% endmermaid %}
+
+We're going to define a producer policy first:
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+    name: producer-policy
+    namespace: ns2
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: server
+      default:
+        idleTimeout: 20s
+```
+
+We know it's a producer policy because it is defined in the same namespace as the `MeshService: server` and names this server in its `spec.to[].targetRef`.
+So both client1 and client2 will receive the timeout of 20s.
```

Comment:
:disappointed: luckily we can ignore this check in general but this `Google.Units` we can probably skip